gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ui/SnappyRegionStatsCollectorFunction.java

```diff
@@ -40,7 +40,11 @@
 
   public static String ID = "SnappyRegionStatsCollectorFunction";
 
-  public static final ObjectSizer sizer = ObjectSizer.getInstance(false);
+  public static final ObjectSizer sizer = ObjectSizer.getInstance(true);
+
+  static {
+    sizer.setForInternalUse(true);
```

Comment:
Previously we were using an existing instance of ObjectSizer. This was logging unnecessary log statements. Created a new instance of ObjectSizer and set it as internal instance so it does not log the unwanted statements.